requirements/base.txt

```diff
@@ -30,3 +30,5 @@ ifcfg==0.17
 sqlalchemy==1.2.10
 semver==2.8.1
 sentry-sdk==0.7.9
+django-redis-cache==2.0.0
+redis==3.2.1
```

Comment:
are there any plans on how to package redis with kolibri in the pex installer?
This decision will affect all the installers deeply.

Reply:
The intention of this PR is simply to make it possible if Redis is available externally (for cloud hosted versions of Kolibri, or for kolibri-server). No plans for bundling yet!